cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -98,7 +98,7 @@ spec:
       # If your OS already sets up this number to a higher value, feel free
       # to remove this init container.
       initContainers:
-      - image: alpine:3.6
+      - image: alpine:3.9
```

Comment:
After creating this PR, alpine 3.10 has been released.
Can we update to the version at this time?